geniza/corpus/urls.py

```diff
@@ -34,4 +34,9 @@
         name="document-annotations",
     ),
     path("export/pgp-metadata-old/", corpus_views.pgp_metadata_for_old_site),
+    path(
+        "admin/corpus/document/merge/",
+        corpus_views.DocumentMerge.as_view(),
+        name="document-merge",
+    ),
```

Comment:
Let's configure the url for this on the admin model instead of doing it here in the main corpus config. Here's the relevant section of the docs:
https://docs.djangoproject.com/en/4.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.get_urls

Reply:
Done in 211b59d 👍